packages/app-staking/src/Overview/CurrentList.tsx

```diff
@@ -32,6 +33,18 @@ class CurrentList extends React.PureComponent<Props> {
     );
   }
 
+  private renderAddress (address: string) {
+    if (keyring.getAccounts()) {
```

Comment:
We should be able to do` keyring.getAccount(address))`and it is not available, horrible :(

There it will fail though since `getAccounts()` should return something. So the first condition is always true.

I suppose (since we don't have the getAccount, and we really should), we can do -

```
let pair = keyring.getAccounts().filter(account => account.address() === address);

if (!pair) {
  pair = keyring.getAddress(address);
}

return pair
  ? pair.getMeta().name
  : address;
```